src/sentences.rs

```diff
@@ -128,19 +139,19 @@ fn get_all_sentences<'r>(
                 structure::text
             FROM sentence
             JOIN language ON (sentence.language_id = language.id)
+            WHERE sentence.id > $1
```

Comment:
your ids are not sequential , so if a new id get inserted you will  certainly miss it.